sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/azure.yaml

```diff
@@ -107,6 +107,10 @@ provider:
       metadata:
         name: Azure Storage Account Keys
       detection:
+        pre:
+          include:
+            content:
+              - "AccountKey"
```

Comment:
This can even be `AccountKey=` to be more specific.

Reply:
very true

```suggestion
              - "AccountKey="
```